Sources/LaunchItem.swift

```diff
@@ -72,101 +93,77 @@ public enum LaunchItem: CustomStringConvertible, Equatable {
     public var description: String {
         // Creating a custom string for LaunchItem to prevent a crash when printing any enum values with associated items on iOS 8. Swift accesses the class of every possible associated object when printing an enum instance with an asssociated value, no matter which case the instance represents. This causes a crash on iOS 8 since swift attempts to access UIApplicationShortcutItem, which doesn't exist on iOS 8. Filed rdar://26699861 – hoping for a fix in Swift 3.
         switch self {
-        case let .RemoteNotificationItem(remoteNotification):
-            return "LaunchItem.RemoteNotificationItem: \(remoteNotification)"
-        case let .LocalNotificationItem(localNotification):
-            return "LaunchItem.LocalNotificationItem: \(localNotification)"
-        case let .OpenURLItem(urlToOpen):
-            return "LaunchItem.OpenURLItem: \(urlToOpen)"
-        case let .ShortcutItem(shortcutItem):
-            return "LaunchItem.ShortcutItem: \(shortcutItem)"
-        case let .UserActivityItem(userActivity):
-            return "LaunchItem.UserActivityItem: \(userActivity)"
-        case .NoItem:
-            return "LaunchItem.NoItem"
+        case let .remoteNotification(item):
+            return "LaunchItem.remoteNotification: \(item)"
+        case let .localNotification(item):
+            return "LaunchItem.localNotification: \(item)"
+        case let .openURL(item):
+            return "LaunchItem.openURL: \(item)"
+        case let .shortcut(item):
+            return "LaunchItem.shortcut: \(item)"
+        case let .userActivity(item):
+            return "LaunchItem.userActivity: \(item)"
+        case .none:
+            return "LaunchItem.none"
         }
     }
     
     // MARK: Public Properties
     
     /// The launch options that were used to construct the enum, for passing into third party APIs.
-    public var launchOptions: [NSObject : AnyObject] {
+    public var launchOptions: [AnyHashable : Any] {
         get {
```

Comment:
I think `get` is the default assumption if you don't specify it, which would help reduce a level of indentation here. Personally, I don't like `set` blocks and thus never find the need for `get` blocks either.